src/components/tutor-input.cjsx

```diff
@@ -141,7 +141,7 @@ TutorDateInput = React.createClass
           weekStart={0}
         />
     else if @props.disabled and value
-      displayValue = value.toString("YYYY/MM/DD")
+      displayValue = value.toString("MM/DD/YYYY")
 
```

Comment:
Also, I'm guessing the spec or mockup has the dates formatted that way?   I personally dislike that format because you can't readily  tell what 1/2/2016 means.